src/structure/group.rs

```diff
@@ -35,3 +38,45 @@ pub trait Group<O: Op>
 {}
 
 impl_marker!(Group<Additive>; i8, i16, i32, i64);
+
+/*
+ *
+ * A subgroup trait inherit from its parent groups.
+ *
+ */
+
+/// The group `E(n)` of isometries, i.e., rotations, reflexions, and translations.
+pub trait EuclideanGroupApprox<S: RealApprox, E: EuclideanSpaceApprox<S>>: GroupApprox<Multiplicative> {
+    /// Applies this group's action on a point from the euclidean space.
+    fn transform_point(&self, pt: &E) -> E;
+
+    /// Applies this group's action on a vector from the euclidean space.
+    ///
+    /// If `v` is a vector and `a, b` two point such that `v = a - b`, the action `∘` on a vector
+    /// is defined as `self ∘ v = (self × a) - (self × b)`.
+    fn transform_vector(&self, v: &E::Vector) -> E::Vector;
+}
+
+/// The group `SE(n)` of orientation-preserving isometries, i.e., rotations and translations.
+///
+/// This is a subgroup of `E(n)`.
+pub trait SpecialEuclideanGroupApprox<S: RealApprox, E: EuclideanSpaceApprox<S>>: EuclideanGroupApprox<S, E> {
+}
+
+/// The group `T(n)` of translations.
+///
+/// This is a subgroup of `SE(n)`.
+pub trait TranslationGroupApprox<S: RealApprox, E: EuclideanSpaceApprox<S>>: SpecialEuclideanGroupApprox<S, E> {
```

Comment:
Do you have any link for any information that T(n) is subgroup of SE(n)... Cannot find it myself. (Not that I have any doubts, but I just really want to learn more :P)